storage/remote/read_test.go

```diff
@@ -33,13 +33,16 @@ func mustNewLabelMatcher(mt labels.MatchType, name, val string) *labels.Matcher
 	return m
 }
 
+/*
```

Comment:
Need to fix this!

Reply:
looks like getting all of the read code to use `labels.Labels` is going to be a bit of work, can we just turn external labels back into `model.LabelSet` for now and do that in another PR?

Reply:
I updated it to use `labels.Labels`;  we're trying to move away from `model`.